.github/workflows/security.yml

```diff
@@ -0,0 +1,89 @@
+name: Scan container images
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  schedule:
+    - cron: '0 0 * * 1'
```

Comment:
This will run at midnight UTC (5:30 IST). How about running scan at 9:30 IST so it would be convenient for us?

Reply:
this works too